### PR TITLE
Completed Requirements

### PR DESCRIPTION
**Pull request by harishr1308**
Autogen.sh runs successfully

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,35 @@ I did this ages ago, so I don't really remember.
 
 1.  Install `gnome-common`
 2.  Install the `python3-arrow`  and `python3-markdown`
-3.  Install a SCSS compiler (eg. `rubygem-sass` on Fedora).  This is very
+3.  Install `libglib2.0-dev`,`libgtk-3-dev`,`libsoup2.4-dev`,`libjson-glib-dev`
+4.  Install a SCSS compiler (eg. `rubygem-sass` on Fedora).  This is very
```
Comment (Dragnucs) [inline README.md:33]: Other SCSS compilers do work. Please do not recommend a tool that needs installing additional dependencies.

File: README.md [hunk 2]
```diff
@@ -29,9 +29,35 @@ I did this ages ago, so I don't really remember.
 
 1.  Install `gnome-common`
 2.  Install the `python3-arrow`  and `python3-markdown`
```
Comment (samdroid-apps) [inline README.md:31]: While we are updating the deps, this should no longer be needed (the app is now in vala)

File: README.md [hunk 3]
```diff
@@ -29,9 +29,35 @@ I did this ages ago, so I don't really remember.
...
-3.  Install a SCSS compiler (eg. `rubygem-sass` on Fedora).  This is very
+3.  Install `libglib2.0-dev`,`libgtk-3-dev`,`libsoup2.4-dev`,`libjson-glib-dev`
```
Comment (samdroid-apps) [inline README.md:32]: -dev is the debian name, -devel is the fedora name.  Maybe just keep it distro agnostic and only put the library names there.

File: README.md [hunk 4]
```diff
@@ -29,9 +29,35 @@ I did this ages ago, so I don't really remember.
...
+4.  Install a SCSS compiler (eg. `rubygem-sass` on Fedora).  This is very
     important because otherwise it will fail to build.
-
+    For Ubuntu run the following commands :
```
Comment (samdroid-apps) [inline README.md:35]: Can we move the distro specific stuff to the end?

File: README.md [hunk 5]
```diff
@@ -29,9 +29,35 @@ I did this ages ago, so I don't really remember.
...
+    `sudo apt-get install rubygems ruby-dev`
+    `sudo su -c "gem install sass"`
+    `sudo gem install sass`
+5.  Install the package `gobject_introspection` by using the command `sudo apt install gobject-introspection`
+
+Installing libglib2.0_50
```
Comment (samdroid-apps) [inline README.md:41]: Wow, this is a hard way to do it.  Right now, I'm just using GNOME's `jhbuild` tool to install everything.  It might help you too!

**Review by samdroid-apps — CHANGES_REQUESTED** [review]
Thanks for the contribution!  Feel free to ask questions if you have any more.  Sorry that I went head-first into the re-write without updating these docs.

File: README.md (hunk 4 above)
Comment (harishr1308) [reply README.md:35]: Yea sure go ahead

File: README.md (hunk 5 above)
Comment (harishr1308) [reply README.md:41]: oh great that simplifies stuff on my end as well

**Comment by samdroid-apps** [comment]
I think this is no longer needed as we went back to python.  Feel free to test and tell me if there are still issues!  Closing this PR right now though.

**Pull request closed without merging**